docs/release-notes/2.40.0.md

```diff
@@ -0,0 +1,126 @@
+
+# Percona Monitoring and Management 2.40.0
+
+
+| **Release date:** | [Oct 02, 2023]                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+<!---
+
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+
+--->
+
+## Release Highlights
+
+### Inventory improvements
+
+#### Ability to update labels for existing services to PMM  
+
+Starting with PMM 2.40.0, editing labels for existing services is easier and much more efficient. You can edit a service label directly from the PMM UI without removing and re-adding it. For more information on editing labels, see [documentation](https://docs.percona.com/percona-monitoring-and-management/details/dashboards/dashboard-inventory.html#editing-labels-for-a-service).
+
+  ![!](../../_images/PMM_access_edit_labels.png)
+
+Furthermore, in our continued effort  to enhance user experience, we have redesigned the following pages to make them more user-friendly and intuitive:
+
+- Select service type page
+- Add service page
+
+
+#### Connecting Services and Nodes 
+
+Starting with PMM 2.40.0, you can click on the link in the **Node Name** column to view the node on which a specific service is running and analyze how node-level resource utilization impacts the performance of those services.
+
```

Comment:
This also links from the Node page to Services running on this Node. will be good to describe and image

Reply:
Done.